src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java

```diff
@@ -66,8 +66,8 @@ public class EOtext$EOreplaced extends PhDefault {
             new AtComposite(
                 this,
                 rho -> {
-                    final String target = new Dataized(rho.attr("target").get())
-                        .take(String.class);
+                    final String target = (new Dataized(rho.attr("target").get())
+                        .take(String.class)).replaceAll("\\$", "\\\\\\$");
```

Comment:
@EugeneDar I think I would be good to leave `@todo` here with task to decide what to do with other special symbols and than do smth with them.

Reply:
@EugeneDar WDYT?

Reply:
@levBagryansky I fixed this